### PR TITLE
Make date switcher work

### PR DESCRIPTION
**Pull request by ElisabethCardella**

### CODE REVIEW

File: src/components/DateBox/index.js [hunk 1]
```diff
@@ -1,8 +1,36 @@
...
+  isToday,
+  isTomorrow,
+  isYesterday,
+  sub,
+} from "date-fns";
+
+const DateBox = () => {
+  const [date, setDate] = useState(new Date());
+
+  console.log(date);
```
Comment (marcingardas) [inline src/components/DateBox/index.js:18]: Remove `console.log`?

File: src/components/DateBox/index.js [hunk 2]
```diff
@@ -1,8 +1,36 @@
...
+      return "Today";
+    }
+    if (isYesterday(date)) {
+      return "Yesterday";
+    }
+    if (isTomorrow(date)) {
+      return "Tomorrow";
+    }
+
+    return format(date, "MM/dd/yyyy");
```
Comment (marcingardas) [inline src/components/DateBox/index.js:31]: "MM/dd/yyyy" is not the format mentioned in the Jira ticket, can you update?

File: src/helpers/dateFunctions.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+import { formatDistance, subDays } from 'date-fns';
```
Comment (marcingardas) [inline src/helpers/dateFunctions.js:1]: Empty file can be removed?

**Review by marcingardas — CHANGES_REQUESTED** [review]
Looks good! Just a couple of small comments

File: src/components/DateBox/index.js (hunk 1 above)
Comment (ElisabethCardella) [reply src/components/DateBox/index.js:18]: doneeeeeeeeeeeeeeeeeeeee

File: src/components/DateBox/index.js (hunk 2 above)
Comment (ElisabethCardella) [reply src/components/DateBox/index.js:31]: donnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnne

File: src/helpers/dateFunctions.js (hunk 3 above)
Comment (ElisabethCardella) [reply src/helpers/dateFunctions.js:1]: done

**Review by marcingardas — APPROVED** [review]
Great work 💪

**Pull request merged**